src/PrestaShopBundle/Command/ConfigCommand.php

```diff
@@ -0,0 +1,320 @@
+    }
+
+    /**
+     * Get and print out one configuration value
+     */
+    private function get(): void
+    {
+        $key = $this->input->getArgument('key');
+
+        if (strpos($key, '*') !== false) {
+            if (!$this->idLang) {
+                throw new Exception('Using wildcards requires --lang option', self::STATUS_LANG_REQUIRED);
+            }
+            try {
+                $keys = array_filter($this->configuration->keys(), function (string $v) use ($key) {
+                    return fnmatch($key, $v);
+                });
+            } catch (NotImplementedException $e) {
+                throw new Exception('Not implemented yet', self::STATUS_NOT_IMPLEMENTED);
+            } catch (Exception $e) {
+                $msg = sprintf('Could not load all configuration keys: %s', $e->getMessage());
+                throw new Exception($msg, self::STATUS_ERROR);
+            }
+        } else {
+            $keys = [$key];
+        }
+
+        foreach ($keys as $key) {
+            $value = $this->configuration->get($key, null, $this->shopConstraint);
+
+            // non language values will be just strings
+            // but for language dependant values the response is an array
+            if (is_array($value)) {
+                if (!$this->idLang) {
+                    $msg = sprintf('%s is language dependant, --lang option is required', $key);
+                    throw new Exception($msg, self::STATUS_LANG_REQUIRED);
+                }
+                $value = $value[$this->idLang];
+            }
+
+            $this->io->success(sprintf('%s=%s', $key, $value));
+        }
+    }
+
+    /**
+     * Set and print out one configuration value
+     */
+    private function set(): void
+    {
+        $key = $this->input->getArgument('key');
+
+        if (strpos($key, '*') !== false) {
+            throw new Exception('Set action does not support wildcards', self::STATUS_INVALID_OPTIONS);
+        }
+
+        $newvalue = $this->input->getOption('value');
+
+        // new value is required for set
+        if (is_null($newvalue)) {
+            throw new Exception('Value required for action "set"', self::STATUS_VALUE_REQUIRED);
+        }
+
+        // make the value language array if lang is set
+        if (!is_null($this->idLang)) {
+            $newvalue = [
+                $this->idLang => $newvalue,
+            ];
+        }
+
+        // set the value
+        try {
+            $this->configuration->set($key, $newvalue, $this->shopConstraint);
+        } catch (\Exception $e) {
+            $msg = sprintf('Failed setting value: %s', $e->getMessage());
+            throw new Exception($msg, self::STATUS_FAILED_SET);
+        }
+
+        $this->get();
+    }
+
+    /**
+     * Remove one configuration value
+     */
+    private function remove(): void
+    {
+        $key = $this->input->getArgument('key');
+
+        if (strpos($key, '*') !== false) {
```

Comment:
Same as previous use `Validate::isConfigName($key)`

Reply:
Don't add behavior you don't need, just "in case of". If the Config name is not valid, raise and exception, don't go to far

Reply:
Dropped wildcard support in 8ee6e31